ai_engine_sdk/client.py

```diff
@@ -289,22 +300,35 @@ async def get_private_function_groups(self) -> List[FunctionGroup]:
             method='GET',
             endpoint="/v1beta1/function-groups/"
         )
-
         return list(
             map(
                 lambda item: FunctionGroup.parse_obj(item),
                 raw_response
             )
         )
 
-    async def get_credits(self) -> CreditBalance:
-        response = await make_api_request(self._api_base_url, self._api_key, 'GET', "/v1beta1/engine/credit/info")
-        return CreditBalance(
-            totalCredits=response['total_credit'],
-            usedCredits=response['used_credit'],
-            availableCredits=response['available_credit']
+    async def create_function_group(
+            self,
+            is_private: bool,
+            name: str
+    ) -> FunctionGroup:
+        payload = {
+            "isPrivate": is_private,
```

Comment:
this is a good point here. Currently only private FG-s can be created, the api won't allow public one.

Reply:
Tbh, I did not notice it... 
There is any business/logic reason to restrict other than internal ppl to create public function groups? 
![image](https://github.com/user-attachments/assets/bbdc0df0-85ca-4a9c-aefe-0196f38ce4f2)